README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
Good point.

I think this is highlighting the fact that making a distinction between a prefix and a full address is a bit counter-productive.

If I don't distinguish between these two concepts, everything is great. I can quote for a prefix or an address. The implementation providing the quote just finds the longest matching prefix in their routing table and - if local - returns it or - if remote - recurses.

Similarly, when I receive a response, I don't have to care if it's a prefix or an account. It could be an account, i.e. "this quote is only valid if you send to this exact account." Or it could be a prefix. Either way, all I need to do is check whether my destination starts with this and if so, the quote applies.

Most of the time, you'll quote for an account and it'll apply to a whole prefix, but there could be exceptions.

It's confusing due to the fact that we distinguish between accounts and prefixes, which is a manufactured problem.

Not sure what to do about it - my initial reaction is to use the field names we have and in the spec write the type as "address or prefix" for both and explain that it'll usually be accounts for requests and prefixes for responses, but that you should test all four cases.